formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Decoder.kt

```diff
@@ -426,9 +426,11 @@ internal class CborParser(private val input: ByteArrayInput, private val verifyO
             } else {
                 val header = curByte and 0b111_00000
                 val length = elementLength()
-                if (header == HEADER_ARRAY || header == HEADER_MAP) {
+                if (header == HEADER_TAG) {
+                    readNumber()
+                } else if (header == HEADER_ARRAY || header == HEADER_MAP) {
```

Comment:
cc @JesusMcCloud as author of these changes. I do not see any particular problems, so I'll merge this if you don't have other comments.

Reply:
Thanks! Looks good to me!